File: Selenium/Project/Activity6.py
# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager
from selenium.webdriver.support.wait import WebDriverWait

# Set up the Firefox Driver with WebDriverManger
service = FirefoxService(GeckoDriverManager().install())

# Start the Driver
with webdriver.Firefox(service=service) as driver:
     # Navigate to the URL
    driver.get("https://alchemy.hguy.co/lms/")

    wait = WebDriverWait(driver, 10)

    My_Account = driver.find_element(By.XPATH, "//a[contains(text(),'My Account')]")
    My_Account.click()
    Page_tittle = driver.title
    print("Title of the page : ", Page_tittle)
    driver.find_element(By.XPATH, "//*[contains(text(),'Login')]").click()
    driver.find_element(By.ID, "user_login").send_keys("root")
    driver.find_element(By.ID, "user_pass").send_keys("pa$$w0rd")
    driver.find_element(By.ID, "wp-submit").click()
    Display_Username =  driver.find_element(By.CLASS_NAME, "display-name").text
    print("User has been logged in")
    assert Display_Username == "root"

File: Selenium/Project/Activity6.java
package project;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

import io.github.bonigarcia.wdm.WebDriverManager;

public class Activity6 {
	
	WebDriver driver;	
	  @BeforeClass
	    public void beforeClass() {
	        // Set up the Firefox driver
	        WebDriverManager.firefoxdriver().setup();
	        driver = new FirefoxDriver();
	 
	        //Open browser
	        driver.get("https://alchemy.hguy.co/lms/");
	        
	    }
	  
	  @Test
	    public void My_Account_and_Login_Page()
      {
		
		  driver.findElement(By.xpath("//*[contains(text(),'My Account')]")).click();
		  String Page_title = driver.getTitle();
		  System.out.println("Title of the Page is : " + Page_title);
		  driver.findElement(By.xpath("//*[contains(text(),'Login')]")).click(); 
		  driver.findElement(By.id("user_login")).sendKeys("root");
		  driver.findElement(By.id("user_pass")).sendKeys("pa$$w0rd");
		  driver.findElement(By.id("wp-submit")).click();
		  String Display_Username =  driver.findElement(By.className("display-name")).getText();
		  Assert.assertEquals("root", Display_Username );
      }

	  @AfterClass
	    public void afterClass() {
	        //Close browser
	        driver.close();
	    }
}
